packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -39,10 +43,17 @@ export type WebSocketEvent = {
 /**
  * Represents the response from getUserUsingCI method.
  *
- * @public
  */
 
 export type AgentResponse = {
+  /**
+   * id of the agent.
+   */
+  id: string;
+  /**
+   * The user id of the agent.
```

Comment:
Maybe we should find a better description for this. Is this the User ID in Webex?

Reply:
addressed